packages/core/src/icon/icon.performance.ts

```diff
@@ -36,7 +36,7 @@ describe('cds-icon performance', () => {
       loadTextEditIconSet();
       loadTravelIconSet();
     `;
-    expect((await testBundleSize(bundle)).kb).toBeLessThan(104);
+    expect((await testBundleSize(bundle)).kb).toBeLessThan(111.5);
```

Comment:
Some of these went up because perf tests are not as aggressive minifying/removing comments to cut the overall test time in half.